context.go

```diff
@@ -130,79 +129,61 @@ var testContextData = struct {
 }{service: make(map[string][]byte, 0)}
 
 // Save takes an identifier and an interface. The interface will be network.Marshaled
-// and saved under a filename based on the identifier. An eventual error will be returned.
-// If contextDataPath is non-empty, the destination is a file: it will be created
-// with rw-r----- permissions (0640). If the file already exists, it will be overwritten.
-//
-// The path to the file is chosen as follows:
```

Comment:
Kind of too bad to lose this documentation of where the files are... did it move someplace else?

Reply:
The Save function doesn't create the files anymore, so I removed this bit of the comment. But it's still relevant for `initContextDataPath()`, so I'll add it back there